src/OrchardCore.Modules/OrchardCore.Users/Controllers/AdminController.cs

```diff
@@ -316,7 +316,7 @@ public async Task<IActionResult> Create()
 
         [HttpPost]
         [ActionName(nameof(Create))]
-        public async Task<IActionResult> CreatePost()
+        public async Task<IActionResult> CreatePost([Bind(Prefix = "User.Password")] string password)
```

Comment:
why `User` prefix?

Reply:
The `UserDisplayDriver` build the form with the `User` prefix. So in order for the binding engine to bind the password correctly, we have to use the same prefix. Is there a better way to do this?